.travis.yml

```diff
@@ -4,3 +4,19 @@ node_js:
   - '0.10'
   - '0.12'
   - 'iojs'
+
+cache:
+  directories:
+    - node_modules
+
+matrix:
+  fast_finish: true
```

Comment:
I don't really see the point of fast finish unless you have a *really* slow build, as I usually want to know all the platforms something fails on, not just the first.